modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/information-banner/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+/* eslint-disable no-undef */
+
+const closeButton = fragmentElement.querySelector('.close');
+const bannerElement = document.querySelectorAll('.cp-information-banner');
+const bannerFragment = fragmentElement.querySelector('.cp-information-banner');
+
+const currentDate = new Date();
+const strData = configuration.expirationDate;
```

Comment:
what does `str` stand for? can you rename the variable to be a bit more clear

Reply:
Done.